cmd/opampsupervisor/supervisor/supervisor_test.go

```diff
@@ -91,6 +91,12 @@ telemetry:
               endpoint: http://localhost:14317
   logs:
     level: info
+    processors:
+      - batch:
+          exporter:
+            otlp:
+              protocol: http/protobuf
+              endpoint: https://backend:4317
```

Comment:
does this test end up erroring out trying to emit logs to this non existent backend?

Reply:
only asking because i ran into this when trying to send logs to a non existent backend before

Reply:
No, it only tests that the proper values are not nil, it doesn't try to execute them.